logstash-core/lib/logstash/util/wrapped_acked_queue.rb

```diff
@@ -332,10 +332,6 @@ def flush_signal_received?
 
       private
 
-      def iterating?
-        @iterating
-      end
```

Comment:
@guyboertje 

> Maybe we should adopt Hungarian prefix b_ for booleans only in LS Ruby code? Half joking on here.

Yea def, for Ruby (since we have no types) some convention on this kind of variable does make sense imo. As a Java person I'm more used to `is_` prefixes (without the underscore in Java) for this, but `b_` I like too :)